vizro-core/examples/visual-vocabulary/app.py

```diff
@@ -108,6 +110,15 @@ def make_navlink(chart_group: ChartGroup) -> vm.NavLink:
 )
 
 app = Vizro().build(dashboard)
+app.dash.layout.children.append(
+    dbc.NavLink(
+        ["Made with ", html.Img(src=get_asset_url("logo.svg"), id="banner", alt="Vizro logo"), "vizro"],
+        href="https://github.com/mckinsey/vizro",
+        target="_blank",
+        external_link=True,
```

Comment:
Not important but FYI this isn't really necessary since it's set automatically when `href` is an absolute URL.
```suggestion
```